consul/datadog_checks/consul/data/conf.yaml.example

```diff
@@ -45,7 +45,7 @@ instances:
 
     ## @param ca_bundle_file - string - optional - default: true
     ## Whether to verifiy SSL certificates for HTTPS requests.
-    ## Possible values: True, False or '<TRUSTED_CA_BUNDLE_FILE_PATH>'
+    ## Possible values: `true`, `true` or '<TRUSTED_CA_BUNDLE_FILE_PATH>'
```

Comment:
Shouldn't this be `true`, `false` [...] ?

Reply:
I'm not sure to understand. Could you elaborate ? : )

Reply:
Oh, right, saw the typo, thanks.

Here is the PR to fix it: https://github.com/DataDog/integrations-core/pull/4033